docs/subnets/reference-elastic-subnets-parameters.md

```diff
@@ -0,0 +1,255 @@
+$$
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+
+The larger $StakingPeriod$ is, the closer $EffectivePeriod$ is to $MaxConsumptionRate$.
+
+A staker achieves the maximum reward for its stake if $StakingPeriod$ = $Minting Period$.
+The reward is:
+
+<!-- markdownlint-disable MD013 -->
+<!-- vale off -->
+$$
+Max Reward = \left(MaximumSupply - Supply \right) \times \frac{Stake}{Supply} \times \frac{MaxConsumptionRate}{PercentDenominator}
+$$
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+
+Note that the reward formula above is used in the Primary Network to calculate
+stakers reward. For reference, you can find Primary network parameters in [the section below](#primary-network-parameters-on-mainnet).
+
+## Delegators Weight Checks
+
+There are bounds set of the maximum amount of delegators' stake that a validator can receive.
+
+The maximum weight $MaxWeight$ a validator $Validator$ can have is:
+
+<!-- markdownlint-disable MD013 -->
+<!-- vale off -->
+$$
+    MaxWeight = \min(Validator.Weight \times MaxValidatorWeightFactor, MaxValidatorStake)
+$$
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+
+where $MaxValidatorWeightFactor$ and $MaxValidatorStake$ are the Elastic Subnet
+Parameters described above.
+
+A delegator won't be added to a validator if the combination of their weights
+and all other validator's delegators' weight is larger than $MaxWeight$. Note
+that this must be true at any point in time.
+
+<!-- vale off -->
+Note that setting $MaxValidatorWeightFactor$ to 1 disables delegation since the $MaxWeight = Validator.Weight$.
+<!-- vale on -->
+
+## Notes on Percentages
+
+`PercentDenominator = 1_000_000` is the denominator used to calculate percentages.
+
+It allows you to specify percentages up to 4 digital positions.
+To denominate your percentage in `PercentDenominator` just multiply it by `10_000`.
+For example:
+
+* `100%` corresponds to `100 * 10_000 = 1_000_000`
+* `1%` corresponds to `1* 10_000 = 10_000`
+* `0.02%` corresponds to `0.002 * 10_000 = 200`
+* `0.0007%` corresponds to `0.0007 * 10_000 = 7`
+
+## Primary Network Parameters on Mainnet
+
+An Elastic Subnet is free to pick any parameters affecting rewards, within the
+constraints specified above. For reference we list below Primary Network
+parameters on Mainnet:
+
+* `AssetID = Avax`
+* `InitialSupply = 240_000_000 Avax`
+* `MaximumSupply = 720_000_000 Avax`.
+* `MinConsumptionRate = 0.10 * reward.PercentDenominator`.
+* `MaxConsumptionRate = 0.12 * reward.PercentDenominator`.
+* `Minting Period = 365 * 24 * time.Hour`.
+* `MinValidatorStake = 2_000 Avax`.
+* `MaxValidatorStake = 3_000_000 Avax`.
+* `MinStakeDuration = 2 * 7 * 24 * time.Hour`.
+* `MaxStakeDuration = 365 * 24 * time.Hour`.
+* `MinDelegationFee = 20000`, that is `2%`.
+* `MinDelegatorStake = 25 Avax`.
+* `MaxValidatorWeightFactor = 5`. This is a platformVM parameter rather than a
+  genesis one, so it's shared across networks.
+* `UptimeRequirement = 0.8`, that is `80%`.
```

Comment:
Adding link to the code at https://github.com/ava-labs/avalanchego/blob/master/genesis/genesis_mainnet.go#L20 would be useful too.

Reply:
I thought about it but not all parameters are in genesis. And keeping the reference up to date may be challenging. Should I really go for it?

Reply:
k, then. Thanks.